stl/inc/yvals_core.h

```diff
@@ -983,7 +983,18 @@
 #define _CXX20_DEPRECATE_MOVE_ITERATOR_ARROW
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4032
+#if !defined(_SILENCE_NONFLOATING_COMPLEX_DEPRECATION_WARNING)
+#define _DEPRECATE_NONFLOATING_COMPLEX                                                  \
+    [[deprecated("warning STL4032: "                                                    \
+                 "The effect of instantiating the template complex for any type other " \

```

Comment:
I think this should say `std::complex` for consistency with our other deprecation warnings.